js/main.js

```diff
@@ -0,0 +1,31 @@
+
+function getRandomNumber(min, max) {
+  if (max < min) {
+    [min, max] = [max, min];
+  }
+
+  if (min < 0 || max < 0) {
+    return NaN;
```

Comment:
Это сложный вопрос, скорее всего ответ `NaN` будет вообще не очевидным результатом, рассмотрим логически: ты запускаешь функцию чтобы получить случайное число и потом выполняешь операции с этим результатом: 
```ts
const test = getRandomNumber(someNumber, someNumber2)
const test2 = test + 33
//...
const test56 = test2 + 3333
```
Ошибки никакой нет и ты где-то потом видишь, что у тебя значение test56 это NaN, чтобы понять где ошибка тебе придется идти вверх по всем строкам и искать где же произошла ошибка, на каком этапе

Reply:
Наиболее адекватный вариант (на мой взгляд) остановить и выбросить свою ошибку, например
```suggestion
    throw new RangeError('min or max must ne more then 0');
```
Поскольку выброс исключений вы ещё не проходили, есть вариант просто написать в консоль и завершить выполнение вернув undefined
```suggestion
    console.error('min or max must ne more then 0');
    return
```

Reply:
Или всеми силами попытаться выполнить эту функцию, например привести отрицательные к положительным и т.д., но на мой взгляд, это тоже плохой вариант, поскольку результат опять получится верным и код продолжит выполняться, даже с "неверными" аргументами

Reply:
В любом случае это не ошибка, я просто тебе рассказываю, какие есть ещё решения, помимо как вернуть NaN и какие подводные камни